src/Framework/test/TargetingPackTests.cs

```diff
@@ -102,15 +102,22 @@ public void RefAssemblyReferencesHaveExpectedAssemblyVersions()
 
             Assert.All(dlls, path =>
             {
-                using var fileStream = File.OpenRead(path);
-                using var peReader = new PEReader(fileStream, PEStreamOptions.Default);
-                var reader = peReader.GetMetadataReader(MetadataReaderOptions.Default);
-                
-                Assert.All(reader.AssemblyReferences, handle =>
+                // Skip netstandard2.0 System.IO.Pipelines assembly. References have old versions.
+                var filename = Path.GetFileName(path);
+                if (!string.Equals("System.IO.Pipelines.dll", filename, StringComparison.OrdinalIgnoreCase))
                 {
-                    var reference = reader.GetAssemblyReference(handle);
-                    Assert.Equal(0, reference.Version.Revision);
-                });
+                    using var fileStream = File.OpenRead(path);
+                    using var peReader = new PEReader(fileStream, PEStreamOptions.Default);
+                    var reader = peReader.GetMetadataReader(MetadataReaderOptions.Default);
+
+                    Assert.All(reader.AssemblyReferences, handle =>
+                    {
+                        var reference = reader.GetAssemblyReference(handle);
+                        var result = 0 == reference.Version.Revision;
+
+                        Assert.True(result, $"In {filename}, {reference.GetAssemblyName()} has unexpected version {reference.Version}.");
```

Comment:
Did this to help determine which assembly and reference was failing. Seems useful going forward.